.github/actions/publish-image/action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
   ROLE_TO_ASSUME:
     description: 'OIDC Role to Assume'
     required: true
+  FORCE_TAG:
```

Comment:
An alternative approach would be to have a flag to also push to latest. If that tag exists then we push the image with the tag specified, and push to `latest`. I know this will cause a slight issue with the client teams having to change their tag, but docker standards is to use the `latest` tag and this would be more reusable by other teams.

Reply:
i dont' think its a big problem to have them use `latest` instead of `dev_server-latest`.

Reply:
Then we can all share code